tests/e2e/event_handler/test_paths_ending_with_slash.py

```diff
@@ -99,5 +103,6 @@ def test_alb_url_trailing_slash(alb_multi_value_header_listener_endpoint):
                 method="POST",
                 url=url,
                 json={"body": body},
+                auth=iam_auth(url, "lambda"),
```

Comment:
please use keyword argument so we can easily tell what "lambda" is in this context.

`iam_auth(url=url, aws_service="lambda")`